src/shapes/Text/constants.ts

```diff
@@ -63,7 +63,7 @@ export const styleProperties: Readonly<StylePropertiesType[]> = [
 // regexes, list of properties that are not suppose to change by instances, magic consts.
 // this will be a separated effort
 export const textDefaultValues: Partial<TClassProperties<Text>> = {
-  _reNewline: /\r?\n/,
+  _reNewline: reNewline,
   _reSpacesAndTabs: /[ \t\r]/g,
   _reSpaceAndTab: /[ \t\r]/,
   _reWords: /\S+/g,
```

Comment:
thos imho should be all const. They don't seem fit for class properties

Reply:
I agree